integration-tests/docker-compose.anvil.yml

```
@@ -0,0 +1,40 @@
+services:
+  anvil:
+    image: ghcr.io/foundry-rs/foundry:latest
+    platform: linux/amd64
+    command:
+      - anvil --host 0.0.0.0
+    ports:
+      - 8545:8545
+
+  deploy-sntv2:
+    platform: linux/amd64
+    environment:
+      - API_KEY_ETHERSCAN="" # env var is required, but value isn't used
+    depends_on:
+      - anvil
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        github_org: status-im
+        github_repo: status-network-token-v2
+    command: ["forge", "script", "script/Deploy.s.sol", "--broadcast", "--fork-url", "http://anvil:8545", "--private-key=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"]
```

Long lines like this are hard to follow:
```suggestion
    command: |
        forge script script/Deploy.s.sol
            --broadcast
            --fork-url=http://anvil:8545
            --private-key=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```
Same for the other line.